deployment/common.build.sh

```diff
@@ -23,8 +23,7 @@ get_version()
 (
     local ROOT=${1-$DEFAULT_ROOT}
     grep "AssemblyVersion" ${ROOT}/SharedVersion.cs \
-        | sed -E 's/\[assembly: ?AssemblyVersion\("([0-9\.]+)"\)\]/\1/' \
-        | sed -E 's/.0$//'
+        | sed -E 's/\[assembly: ?AssemblyVersion\("([0-9\.]+)"\)\]/\1/' 
```

Comment:
@joshuaboniface @EraYaN 
Would this break anything in the packaging?
As far as i tested the debian packages build just fine with this change.

Reply:
Aah no this is a holdover from the days of 10.0.2.0 versioning in SharedVersion.cs

Reply:
Yup, interestingly this doesn't break Debian though, but it can certainly be removed here.